system/Session/Handlers/BaseHandler.php

```diff
@@ -105,26 +105,19 @@ abstract class BaseHandler implements SessionHandlerInterface
      */
     protected $ipAddress;
 
-    public function __construct(AppConfig $config, string $ipAddress)
+    public function __construct(SessionConfig $session, string $ipAddress)
```

Comment:
That's a pretty major change.

Reply:
Yes and no. Almost no one writes their own session handlers, so in this particular case, the impact would be minimal. Although, I haven't thought of any other possible problems yet.